lib/maxmind-db-downloader.js

```diff
@@ -95,7 +95,9 @@ var MaxmindDbDownloader = function () {
               }, function (err) {
                 // download resulted in an error, do not rename
                 // remove temp file
-                fs.unlinkSync(targetFilePathTemp);
+                if (fs.existsSync(targetFilePathTemp)) {
```

Comment:
I believe `fs.existsSync` is being [deprecated](https://nodejs.org/api/fs.html#fs_fs_existssync_path). Could use `fs.statSync`, and catch error if file doesn't exists.